src/style.css

```
@@ -1 +1,38 @@
-/* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.star-block {
+  display: flex;
+  flex-direction: row;
+
+}
+
+.stars__star {
+  background-image: url(Star.png);
+  width: 16px;
+  height: 16px;
+  flex-shrink: 0;
```

looks also redundant

---

to fix tests, you can try to add 
```suggestion
  background-repeat: no-repeat;
  background-position: center;
```